src/components/common/NameRequestInfo.vue

```diff
@@ -104,7 +104,7 @@
               class="pt-0 mt-0"
               v-model="hasNameTranslation"
               id="name-translation-checkbox"
-              @click.native="confirmNameTranslation()"
```

Comment:
This is not supported in Vue3 but I tested this and this code works correctly without it.